source/includes/write/transaction.kt

```diff
@@ -0,0 +1,54 @@
+import com.mongodb.*
+import com.mongodb.client.ClientSession
+import com.mongodb.client.MongoClients
+import com.mongodb.client.MongoCollection
+import org.bson.Document
+
+fun main() {
+// start-transaction
+    // Creates a new MongoClient with a connection string
+    val client = MongoClients.create("<connection string>")
+
+    // Gets the database and collection
+    val restaurantsDb = client.getDatabase("sample_restaurants")
+    val restaurantsCollection: MongoCollection<Document> = restaurantsDb.getCollection("restaurants")
```

Comment:
S: consider using a data class to model this data
```suggestion
    val restaurantsDb = client.getDatabase("sample_restaurants")
    val restaurantsCollection: MongoCollection<Restaurant> = restaurantsDb.getCollection("restaurants")
```